signer/src/storage/postgres.rs

```diff
@@ -1951,6 +2062,88 @@ impl super::DbWrite for PgStore {
 
         Ok(())
     }
+
+    async fn write_sweep_transaction(
+        &self,
+        transaction: &model::SweepTransaction,
+    ) -> Result<(), Error> {
+        // We're doing multiple inserts here so we wrap them in a transaction.
+        let mut tx = self.0.begin().await.map_err(Error::SqlxBeginTransaction)?;
+
+        sqlx::query(
+            "
+            INSERT INTO sweep_transactions (
+                txid
+              , signer_prevout_txid
+              , signer_prevout_output_index
+              , signer_prevout_amount
+              , signer_prevout_script_pubkey
+              , amount
+              , fee
+              , created_at_block_hash
+              , market_fee_rate
+            ) 
+            VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)",
+        )
+        .bind(transaction.txid)
+        .bind(transaction.signer_prevout_txid)
+        .bind(transaction.signer_prevout_output_index as i32)
+        .bind(transaction.signer_prevout_amount as i64)
+        .bind(transaction.signer_prevout_script_pubkey.clone())
+        .bind(transaction.amount as i64)
+        .bind(transaction.fee as i64)
```

Comment:
For integer `as _` casts, we should either use the `from`/`try_from` implementations or if the `as _` cast is always fine, just explain why in a comment.

Reply:
Think I got them all in [d181170](https://github.com/stacks-network/sbtc/pull/710/commits/d1811705b06b1167b0e178f41a92f81e67242b14)